pkg/nuclide/flow/lib/FlowLinter.js

```diff
@@ -71,12 +74,10 @@ function processDiagnostics(diagnostics: Array<Object>, targetFile: string) {
   };
 
   // Filter messages not addressing `targetFile` and merge messages spanning multiple files.
-  var messages = diagnostics.map( (diagnostic) => {
-                  var diagnosticMessages = diagnostic['message'];
-                  return mergeFlowMessages(diagnosticMessages);
-                 }).filter(hasMessageWithPath);
-
-  return messages.map(flowMessageToLinterMessage);
+  return diagnostics
+    .map( (diagnostic) => diagnostic['message'] )
+    .filter(hasMessageWithPath);
```

Comment:
I'm getting a parse error here, due to this extraneous semicolon. Did you actually try running this?